src/app/monitor/components/confidence-line-chart/confidence-line-chart.component.html

```diff
@@ -0,0 +1,2 @@
+<kern-line-chart [data]="dataInput">
+</kern-line-chart>
```

Comment:
why is this wrapped in another component? i know we have it for other charts as well but it doesn't make sense to me

Reply:
I only got it working that way tbh haha. Thought this was required for the modules.